main.tf

```diff
@@ -69,6 +69,7 @@ resource "google_healthcare_fhir_store" "fhir_stores" {
   disable_resource_versioning         = lookup(each.value, "disable_resource_versioning", null)
   enable_history_import               = lookup(each.value, "enable_history_import", null)
   complex_data_type_reference_parsing = lookup(each.value, "complex_data_type_reference_parsing", null)
+  enable_history_modifications        = lookup(each.value, "enable_history_modifications", null)
```

Comment:
Do you know if it automatically uses the beta provider since the field is not available in the GA provider?